### PR TITLE
Add amenity/car_sharing

### PR DESCRIPTION
**Pull request by tas50**
Another handy set of brands.

Signed-off-by: Tim Smith <tsmith@chef.io>

### CODE REVIEW

File: brands/amenity/car_sharing.json [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+  "amenity/car_sharing|stadtmobil": {
+    "match": [
+      "amenity/car_sharing|stadtmobil CarSharing-Station"
+    ],
+    "tags": {
+      "amenity": "car_sharing",
+      "brand": "stadtmobil",
+      "brand:wikidata": "Q2327629",
+      "brand:wikipedia": "en:Stadtmobil",
+      "name": "stadtmobil"
```
Comment (ENT8R) [inline brands/amenity/car_sharing.json]: Should not this name start with a capital letter? And the `brand` too?

Comment (tas50) [reply brands/amenity/car_sharing.json]: Based on their signage and website they use an all lowercase name.

Comment (ENT8R) [reply brands/amenity/car_sharing.json]: But I thought I read somewhere that all names in OSM shoudl start with a capital letter. I just can't find the source anymore... Maybe it was also some kind of Osmose error, does anybody else know more?

Comment (bhousel) [reply brands/amenity/car_sharing.json]: In OSM the name and brand should be as-signed. It's totally ok for them to start with a lowercase letter. Thanks for confirming @tas50

**Pull request merged**